miden-lib/asm/sat/miden_wallet.masm

```diff
@@ -0,0 +1,56 @@
+use.miden::sat::account
+use.miden::sat::tx
+
+#! Add a single asset to an account's vault 
+#!
+#! Stack: [ASSET]
+#! Output: []
+#!
+#! - ASSET is the non-fungible asset of interest.
+export.receive_asset
```

Comment:
I wonder if we should change the calling convention here so that we don't change the depth of the stack. Specifically, it would be:
```
Stack:  [ASSET]
Output: [ASSET]
```

Reply:
hmmm - I think it might be better to replace the `ASSET'` with 4 zeros as it contains some extra info about the account which we may not want to leak. It will require a few more cycles, but I think that's OK.